docs/reference/items.md

```diff
@@ -17,8 +17,9 @@ Directus. For the sake of documentation, we'll use a fictional articles collecti
 
 ::: tip Relational Data
 
-Please see [Relational Data](/reference/introduction#relational-data) and [Field Parameters](/reference/query#fields) to
-learn more.
+The item object contains only physical relationships, which means that no alias relationships are returned here. To
+retrieve _all_ relationships see [Relational Data](/reference/introduction#relational-data) and
+[Field Parameters](/reference/query#fields).
```

Comment:
As far as I know, we don't need the term "physical relationships" anywhere else.
So instead of the proposed text, I would probably just borrow the one from https://docs.directus.io/reference/introduction.html#relational-data:
> Directus only retrieves the fields in your items that explicitly have been requested. Relational data can be retrieved nested by using [the fields parameter](https://docs.directus.io/reference/query.html#fields) in REST, or regular nested queries in GraphQL.

Reply:
that sound's way better but it doesn't seem to be correct actually. when I make a GET request on  `/items/:collection/:pk` I receive all fields without explicitly requesting anyone 🤔

Reply:
Wondering if we should update the mentioned sentence as well 😄 

> Directus only retrieves the **relational** fields in your items that explicitly have been requested. Relational data can be retrieved nested by using [the fields parameter](https://docs.directus.io/reference/query.html#fields) in REST, or regular nested queries in GraphQL.